WordPressKitTests/BlockEditorSettingsServiceRemoteTests.swift

```diff
@@ -171,13 +171,32 @@ extension BlockEditorSettingsServiceRemoteTests {
 // MARK: Editor Global Styles support
 extension BlockEditorSettingsServiceRemoteTests {
 
+    func testFetchBlockEditorSettingsInvalidJSON() {
+        stub(condition: isHost("public-api.wordpress.com") && isPath("/wp-block-editor/v1/sites/1/settings") && containsQueryParams(["context": "mobile"])) { _ in
+            HTTPStubsResponse(jsonObject: ["invalid-json"], statusCode: 200, headers: nil)
+        }
+
+        let waitExpectation = expectation(description: "Block Settings should be successfully fetched")
+        service.fetchBlockEditorSettings { (response) in
+            switch response {
+            case .success(let result):
+                XCTAssertNil(result)
+            case .failure:
+                XCTFail("This payload should parse successfully")
+            }
+            waitExpectation.fulfill()
+        }
+
+        wait(for: [waitExpectation], timeout: 0.3)
+    }
+
     func testFetchBlockEditorSettingsNotThemeJSON() {
         stub(condition: isHost("public-api.wordpress.com") && isPath("/wp-block-editor/v1/sites/1/settings") && containsQueryParams(["context": "mobile"])) { _ in
             fixture(filePath: OHPathForFile(self.blockSettingsNOTThemeJSONResponseFilename, Self.self)!, headers: ["Content-Type": "application/json"])
         }
 
         let waitExpectation = expectation(description: "Block Settings should be successfully fetched")
-        service.fetchBlockEditorSettings(forSiteID: siteID) { (response) in
+        service.fetchBlockEditorSettings { (response) in
```

Comment:
I'd expect SwiftLint to complain about the `()` at some point. But, it should also be something it can autocorrect easily 🤷‍♂️